openmc/deplete/operator.py

```diff
@@ -186,7 +186,7 @@ class Operator(TransportOperator):
         "cutoff": FissionYieldCutoffHelper,
     }
 
-    def __init__(self, geometry, settings, chain_file=None, prev_results=None,
+    def __init__(self, model, chain_file=None, prev_results=None,
```

Comment:
This would be a breaking API change, which I'm not necessarily opposed to but we need to make sure it's communicated to users properly. Perhaps we should explicitly check if the first argument they passed is a `Geometry` object and if so, raise an exception that tells them what they need to do.

Reply:
Resolving this thread since the check is in now